wiki/App-Initialization-Flow.md

```diff
@@ -0,0 +1,72 @@
+        6. If none of these conditions apply, the routed-to page is rendered.
+        - _Note: A previous iteration of these docs, and of the inline documentation in Layout.tsx, describe conditional logic in which scopes set to Ethereum addresses should trigger a call to `initNewTokenChain`, generating a new community for the provided address while rendering a `LoadingLayout`. The relevant code is no longer present in the file, and this behavior should be considered deprecated unless explicitly reintroduced._
+9. If `selectChain()` (`/helpers/chain.ts`) is fired, per step #8:
+    1. If no `chain` argument is passed, the function defaults to a `chain` value set my `app.user.selectedChain`, or else `app.config.defaultChain`.
+        - _Note: This is likely dead code that can be  removed._
+    2. If we do not need to initialize a new chain (i.e. the chain we are switching to has already been initialized and selected), exit the function immediately.
+    3. Globally deinit other active communities via `deinitChainOrCommunity`.
+        - This method triggers a cascade of “deinit” calls, which set various statuses to false, destroy connections to blockchain endpoints, and eentually sets `app.chain` to `null`.
+    4. Thus begins the massive if/then statement determining which chain adapter (controller for chain-specific functionality, i.e. `IChainAdapter`) to lazily import. This relies on `ChainBase`, `Network`, and `ChainType` args from the status route’s `ChainInfo` object.
+        - If ChainBase is Substrate, import the Substrate adapter.
+        - If ChainBase is Cosmos, import the Cosmos adapter.
+        - If Network is Ethereum, import the (ETH) tokenAdapter.
+        - If Network is NEAR or NEARTestnet, import the NEAR adapter.
+        - If Network is Sputnik, import the Sputnik adapter (NEAR + gov).
+        - If Network is Compound, import the Compound adapter (ETH).
+        - If Network is Aave, import the Aave adapter (ETH).
+        - If Network is ERC20 or Axie Infinity, import the (ETH) tokenAdapter.
+        - If Network is ERC721, import the (ETH) NftAdapter.
+        - If Network is SPL, import the Solana adapter.
+        - If ChainBase is Solana, import the Solana adapter.
+        - If ChainBase is Ethereum and ChainType is Offchain, import the “generic” Ethereum adapter.
+        - Otherwise, throw an “invalid chain” error.
+    5. `initServer()` is called the returned `IChainAdapter` instance, which clears local storage and makes a call to `/bulkOffchain`, whose data is used to initialize community content.
+        - _Note: Previous documentation referenced the chainEntities adapter refresh. This line is currently commented out, but lingers in the `initServer()` definition._
```

Comment:
We should remove the dead code / commented out code, and #4025 will further remove parts of the chain event loading code.